Makefile

```diff
@@ -73,6 +74,9 @@ coverage:
 
 lint: flake8 pylint pytype
 
+check_types: clean
```

Comment:
I just noticed this target. Can you rename it to `mypy` to match the one with `pytype` below? Then move it just below the `pytype`target and call it from the `lint` target on line 75 so it runs automatically on every `make lint` call. Like `lint: flake8 pylint pytype mypy`.

Reply:
Done